microcosm_postgres/store.py

```diff
@@ -160,9 +160,19 @@ def search(self, *criterion, **kwargs):
 
         """
         query = self._query(*criterion)
+        query = self._order_by(query)
```

Comment:
The `_order_by` function is a mutator on the query object; by default is does nothing, but a subclass can write it to define whatever order by condition you want.